fixtures/alpine/README.md

```diff
@@ -0,0 +1,17 @@
+# Alpine Docker Image
+
+The image located at `fixtures/alpine/alpine-3.20.3.amd64.tar` is a vendored copy of `docker.io/library/alpine:3.20.3`.
+
+This image is vendored because its download process often fails, which disrupts our Kubernetes Integration Tests.
+
+## Generating the Image
+
+To create the tar file, use the following command:
+
+```
+$ docker save -o alpine-3.20.3.amd64.tar docker.io/library/alpine:3.20.3
```

Comment:
FWIW I don't get the same .tar as you when I do that. I also suspect this would save an arm64 blob if I ran this command as-is.

Reply:
Yes. It saves your default platform. You must pass --platform linux/amd64

Reply:
```shell
$ docker pull --platform linux/amd64 docker.io/library/alpine:3.20.3
> 3.20.3: Pulling from library/alpine
> Digest: sha256:beefdbd8a1da6d2915566fde36db9db0b524eb737fc57cd1367effd16dc0d06d
> Status: Image is up to date for alpine:3.20.3
> docker.io/library/alpine:3.20.3
> 
> What's next:
>     View a summary of image vulnerabilities and recommendations → docker scout quickview docker.io/library/alpine:3.20.3

# Note: docker save doesn't take the --platform flag
$ docker save -o alpine-3.20.3.amd64.tar docker.io/library/alpine:3.20.3
$ shasum -a256 alpine-3.20.3.amd64.tar 
> c8f6acdead29241e5034e5382d3df3902919b8c5edcd24c8b385ba38a3f7759c  alpine-3.20.3.amd64.tar
```

This seems to give me a different hash every time, so it's hard to verify like that. (The image digest is deterministic though.)

Reply:
IIRC, docker images don't follow any particular order when saving layers to disk

Reply:
On [d99891f](https://github.com/gravitational/teleport/pull/48321/commits/d99891fb34952a4854aad33f918c4646ca6f9861), I sourced the files from alpine cdn and added gpg validation.